hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -1,22 +1,57 @@
 define([
+    'common/moment',
     'common/angularMocks',
-    'job-contract/app'
+    'job-contract/app',
+    'job-contract/controllers/contract'
 ], function () {
     'use strict';
 
     describe('ContractCtrl', function () {
-        var ctrl;
+        var ctrl, $rootScope, $scope, $controller;
 
         beforeEach(module('hrjc'));
-        beforeEach(inject(function ($controller, $rootScope) {
-            var $scope = $rootScope.$new();
-            $scope.contract = { id: '1' };
+        beforeEach(inject(function (_$controller_, _$rootScope_) {
+
+          $controller = _$controller_;
+          $rootScope = _$rootScope_;
+
+          $scope = $rootScope.$new();
+
+          $scope.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contractCurrent = [];
+          $scope.$parent.contractPast = [];
+
+          ctrl = $controller('ContractCtrl', {
+            $scope: $scope
+          });
 
-            ctrl = $controller('ContractCtrl', { $scope: $scope });
         }));
 
-        it('example', function () {
-            expect(true).toBe(true);
+        it('should make current contract', function () {
```

Comment:
The test suite is supposed to be self-documenting, using the `describe` and `it` blocks to give meaningful description of what area of the code you are testing, how it is supposed to work, etc.

These "it should make current contract" and "it should make past contract" are not understandable. What does it mean? "What" should make current contract? What "make" does"? When does that should happen? It's unclear from the test.

You can use [this commit](https://github.com/civicrm/civihr/blob/bc6869cefce53efc96b2acbc792022752aa1bd6a/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/components/my-leave-report_test.js) as a reference to what i'm talking about